libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/TreatmentView/TreatmentTable.styles.ts

```diff
@@ -43,7 +50,11 @@ export const TreatmentTableStyles: () => IProcessedStyleSet<ITreatmentTableStyle
         fontSize: 14,
         lineHeight: 14,
         paddingBottom: "10px",
-        paddingLeft: "30px",
+        selectors: {
+          "@media screen and (min-width: 1024px)": {
```

Comment:
should we have this as constant?

Reply:
ummmm, not sure if it makes sense, let me think about it, will change in next PR if it is better. Thanks!